contracts/StreamManager.sol

```diff
@@ -83,6 +90,7 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
     constructor(address _payer) {
         payer = _payer;
         admin = msg.sender;
+        feeAddress = msg.sender;
```

Comment:
@CTAPCKPIM we already had the admin address as the fees address. Why did you decide to change that? What is the purpose of the admin address now?

Reply:
@markholdex 
I misunderstood the role of the administrator address, in which case I will fix it